filebeat/channel/factory.go

```diff
@@ -99,6 +99,9 @@ func (f *OutletFactory) Create(cfg *common.Config, dynFields *common.MapStrPoint
 		fields["prospector"] = common.MapStr{
 			"type": config.Type,
 		}
+		fields["input"] = common.MapStr{
```

Comment:
Could we call this `event.type` instead if we already duplicate the field? Thinking of ECS here.

Reply:
I can change it to `event.type`, I would prefer to not mix concerns of ECS, but since we already add this fields it should be OK.